pkg/hintrunner/hint_test.go

```diff
@@ -184,8 +185,14 @@ func TestWideMul128(t *testing.T) {
 	var dstLow ApCellRef = 1
 	var dstHigh ApCellRef = 2
 
-	lhs := Immediate(*big.NewInt(1).Lsh(big.NewInt(1), 127))
-	rhs := Immediate(*big.NewInt(1<<8 + 1))
+	lhsBytes := uint256.NewInt(1).Lsh(uint256.NewInt(1), 127).Bytes32()
```

Comment:
I think it's clearer as `new(uint256.Int).Lsh(uint256.NewInt(1), 127).Bytes32()` given that `newInt(1)` is instantly overwritten